src/connect-zbt-1/thread/thread-enable.md

```diff
@@ -32,10 +32,9 @@ This guide shows you how to use the Home Assistant Connect&nbsp;ZBT-1 as a dedic
 {% image "/static/img/connect-zbt-1/connect-zbt-1-zha-delete.png" "Migrate your Zigbee network or delete the ZHA integration" %}
 {% stepContent %}
 
-- **Scenario 1**: If you have a new Connect ZBT-1 or did not set up Zigbee Home Automation (ZHA) yet, follow the steps on [forming a new Thread network](/hc/en-us/articles/30589636469533).
-- **Scenario 2**: If you use your existing Connect ZBT-1 to run a Zigbee network, [migrate your Zigbee network](/hc/en-us/articles/26123655295261) to another controller.
+- **Scenario 1**: If you use your existing Connect ZBT-1 to run a Zigbee network, [migrate your Zigbee network](/hc/en-us/articles/26123655295261) to another controller.
```

Comment:
I'd emphasize that these steps are preparatory for switching from Zigbee to Thread which will be documented further below. Right now, I'm confused thinking that these steps will tell me how to switch from Zigbee to Thread. Migrating the Zigbee network to another controller doesn't switch from Zigbee to Thread.

Reply:
Ah! ok. Thx for the details. I misunderstood your comment earlier.